envs/make_envs.sh

```diff
@@ -45,6 +44,32 @@ set -e
 
     # basics
     source activate "$env_name"
+
+    # map three environment variables from JSON file of configuration variables when Pythia conda environment is activated
+    mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+    mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d
+    touch $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+    touch $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
+
+    pathval=$( (echo $PYTHIA_CONFIG) | (awk -F"[,:}]" '{for(i=1;i<=NF;i++){if($i~/'PYTHONPATH'\042/){print $(i+1)}}}' | tr -d '"' | tr -d '[[:space:]]') )
```

Comment:
Nifty! can you leave a comment to that effect here and on the subsequent lines that use this trick?